package.json

```diff
@@ -70,7 +71,7 @@
     "prettier": "^1.19.1",
     "ses": "^0.18.8",
     "stylus": "^0.62.0",
-    "vitepress": "1.0.1"
+    "vitepress": "^1.3.0"
```

Comment:
_question_

Worth pinning vitepress to 1.3.0 exactly?

Reply:
I don't have a preference, but happy to do this

Reply:
The reason I was asking is because I noticed that previously with `^1.0.1`, our cloudflare build process would run `yarn install` which would install vitepress 1.3.0 instead. So I think pinning would give us better control and visibility on which version of vitepress we're running.

Reply:
Agree with your reasoning, amended to pin